i18n/en.json

```diff
@@ -6368,7 +5844,7 @@
   },
   {
     "id": "store.sql_user.update_update.app_error",
-    "translation": ""
+    "translation": "We couldn't update the date of the last update of the user"
```

Comment:
Is this for a user editing a post?

Reply:
No, it is used when a user join or leave a team, in that moment the user UpdateAt field is updated to now.